chain/gssmr/defaults.go

```diff
@@ -23,8 +23,8 @@ var (
 	// DefaultBasePath Default node base directory path
 	DefaultBasePath = string("~/.gossamer/gssmr")
 
-	// DefaultMetricsPort is the metrics server port
-	DefaultMetricsPort = uint32(9876)
+	// DefaultMetricsAddress is the default metrics server listening address.
+	DefaultMetricsAddress = "localhost:9876"
```

Comment:
potato question but this doesn't need the `http://`?

Reply:
:potato: no, this is just the `host:port` for the tcp net listener, [go source code relevant line](https://cs.opensource.google/go/go/+/refs/tags/go1.18:src/net/http/server.go;l=2964) :wink: 

Although you can also use `localhost:http` for example, and Go will understand it as `localhost:80`, but that's another thing.